tests/tests_pytorch/run_standalone_tasks.sh

```diff
@@ -31,9 +31,16 @@ if [ -z "$LOGS" ]; then
     exit 1
 fi
 
+
+cd ../../
+
+# test that ddp can launched as a module (-m option)
+echo "Running ddp example as module"
+python -m examples.convert_from_pt_to_pl.image_classifier_5_lightning_datamodule ${args}
```

Comment:
Reviewer: I'm adding this to keep parity to the previous tests who had a parameterization "as_module=True" to launch the command as `python -m`.